tests/test_units.py

```diff
@@ -58,16 +57,13 @@ def test_dimensionality(self):
         with units.context("hydro"):
             fu = 1 * units.parse_units("kg / m**2 / s")
             tu = 1 * units.parse_units("mm / d")
-            fu.to("mmday")
-            tu.to("mmday")
+            fu.to("mm/day")
+            tu.to("mm/day")
```

Comment:
I'm a bit confused here; Are we testing to see that transforming this twice doesn't raise an error?

Reply:
I blindly modified the weird unit to the good one. The last line is useless, but the other tests the hydro context.

Reply:
Remove the line and we're good?
```suggestion
            fu.to("mm/day")
```
Or should we add an assertion here for completion's sake?